.env

```diff
@@ -9,8 +9,13 @@ FLUTTER_WORKSPACE=discovery_engine_flutter
 DART_VERSION=2.14.4
 FLUTTER_VERSION=2.5.3
 
-ANDROID_TARGETS="arm64-v8a x86_64 x86"
```

Comment:
I've removed this target to improve build times for now, as I don't really know if we need a non 64-bit Intel target, considering that we're e.g. not supporting armv7. Let me know if that is not a good idea.